l10n_es_aeat_mod349/models/mod349.py

```diff
@@ -117,6 +119,34 @@ def _create_349_refund_records(self, refunds):
                      'amount_untaxed': refund.cc_amount_untaxed})
         return True
 
+    def _get_domain(self):
+        self.ensure_one()
+        domain = [('state', 'in', ['open', 'paid']),
+                  ('period_id', 'in', self.periods.ids),
```

Comment:
mmm, no lo creo, prefiero tener los dos filtros, son compatible y el aviso
del checkbox ya lo digo así.

2016-06-03 13:12 GMT+02:00 Pedro M. Baeza <notifications@github.com>:

> In l10n_es_aeat_mod349/models/mod349.py
> <https://github.com/OCA/l10n-spain/pull/367#discussion_r65691744>:
>
> > @@ -117,6 +119,34 @@ def _create_349_refund_records(self, refunds):
> >                       'amount_untaxed': refund.cc_amount_untaxed})
> >          return True
> >
> > +    def _get_domain(self):
> > +        self.ensure_one()
> > +        domain = [('state', 'in', ['open', 'paid']),
> > +                  ('period_id', 'in', self.periods.ids),
>
> Deberías añadir el dominio de los periodos sólo si no es de tipo fecha.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OCA/l10n-spain/pull/367/files/df8b8a5d94757ae1a5a4332bf4a9860aec3feef7#r65691744>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKpVC7d-DCBc4NQHVqwuHBqZOw2LhUBdks5qIAwMgaJpZM4ItbTI>
> .
>



-- 
=========================
Santi Argüeso
Twitter: @santiarg

Web: http://comunitea.com
Twitter: @comutuitea
Tfno: 982814805